taskcluster/fpm_taskgraph/transforms/multi_dep.py

```diff
@@ -0,0 +1,74 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Copied from https://github.com/mozilla-mobile/fenix/blob/d918cfdf3841a17545053dca2e7480c648ab4f95/taskcluster/fenix_taskgraph/transforms/multi_dep.py. I removed the treeherder part.